proposals/0029-cooperative-vector.md

```diff
@@ -548,7 +561,7 @@ enum class DXILMatrixLayout : uint {
 ```
 
 Optimal layouts are opaque implementation specific layouts, the D3D call
-`CooperativeVectorConvertMatrix` can be used to convert the *Matrix* to an
+`ConvertMatrixLayout` can be used to convert the *Matrix* to an
```

Comment:
I'm not king of D3D or even a subject, but to me `ConvertMatrixLayout` says convert between row and column major. Something to indicate that it refers to the optimal or linear algebra layouts might be less confusing. I suspect this ship has sailed though.

Reply:
ConvertToLinearAlgebraMatrixLayout ?
ConvertToOptimizedMatrixLayout ?

Reply:
I should probably not use "To" implying one way conversion.  Since it appears you can express both directions of conversion.

It looks like you could even requeset a row major to column major conversion.  So perhaps ConvertMatrix is just fine.

Reply:
Went with ConvertLinearAlgebraMatrix